scripts/test.sh

```diff
@@ -410,17 +410,8 @@ function unconvert_pass {
   run_for_modules generic_checker run_go_tool "github.com/mdempsky/unconvert" unconvert -v
 }
 
-function ineffassign_per_package {
-  # bash 3.x compatible replacement of: mapfile -t gofiles < <(go_srcs_in_module)
-  local gofiles=()
-  while IFS= read -r line; do gofiles+=("$line"); done < <(go_srcs_in_module)
-
-  # TODO: ineffassign should work with package instead of files
-  run_go_tool github.com/gordonklaus/ineffassign "${gofiles[@]}"
-}
-
 function ineffassign_pass {
-  run_for_modules generic_checker ineffassign_per_package
```

Comment:
Shouldn't ineffassign be enabled as part of golangci? Check `tools/.golangci.yaml`

Reply:
Thanks for the reminder! The `tools/.golangci.yaml` already has the `ineffassign`. But the golangci-lint doesn't support multiple modules so... currently the `make verify-lint` doesn't work.

```bash
# cd ./server
➜  server git:(main) golangci-lint run --config ../tools/.golangci.yaml
lease/lease_queue.go:25:6: exported: type name will be used as lease.LeaseWithTime by other packages, and that stutters; consider calling this WithTime (revive)
type LeaseWithTime struct {
     ^
lease/lessor.go:823:39: unused-parameter: parameter 'dr' seems to be unused, consider removing or renaming it as _ (revive)
func (fl *FakeLessor) SetRangeDeleter(dr RangeDeleter) {}
                                      ^
lease/lease.go:119:6: exported: type name will be used as lease.LeaseItem by other packages, and that stutters; consider calling this Item (revive)
type LeaseItem struct {
     ^
lease/lease_queue.go:31:6: exported: type name will be used as lease.LeaseQueue by other packages, and that stutters; consider calling this Queue (revive)
type LeaseQueue []*LeaseWithTime
```

May I fix the lint one by one and then fix the `make verify-lint` after that? The small changeset is easy to be reviewed.

Reply:
Oh, that's unfortunate. Fixing golangci sounds good to me. Thanks for investigating this!

Reply:
After fixing all the lint's issues, I will fix golangci-lint issue.

Reply:
Can you file an issue about golangci being broken so we can track fixes to lint issues in one place?

Reply:
Filed https://github.com/etcd-io/etcd/issues/16610 .